src/main/scala/com/gu/scanamo/Scanamo.scala

```diff
@@ -80,21 +100,38 @@ object Scanamo {
     * >>> val putResult = Scanamo.putAll(client)("farmers")(List(
     * ...   Farmer("Boggis", 43L, Farm(List("chicken"))), Farmer("Bunce", 52L, Farm(List("goose"))), Farmer("Bean", 55L, Farm(List("turkey")))
     * ... ))
-    * >>> Scanamo.getAll[String, Farmer](client)("farmers")('name -> List("Boggis", "Bean"))
+    * >>> Scanamo.getAll[Farmer](client)("farmers")(UniqueKeys(KeyList('name, List("Boggis", "Bean"))))
+    * List(Valid(Farmer(Boggis,43,Farm(List(chicken)))), Valid(Farmer(Bean,55,Farm(List(turkey)))))
+    * }}}
+    * or with some added syntactic sugar:
+    * {{{
+    * >>> import com.gu.scanamo.syntax._
+    * >>> Scanamo.getAll[Farmer](client)("farmers")('name -> List("Boggis", "Bean"))
     * List(Valid(Farmer(Boggis,43,Farm(List(chicken)))), Valid(Farmer(Bean,55,Farm(List(turkey)))))
     * }}}
+    * You can also retrieve items from a table with both a hash and range key
+    * {{{
+    * >>> import com.gu.scanamo.syntax._
+    * >>> import com.amazonaws.services.dynamodbv2.model.ScalarAttributeType._
+    *
+    * >>> case class Doctor(actor: String, regeneration: Int)
+    * >>> val doctorsTableResult = LocalDynamoDB.createTable(client)("doctors")('actor -> S, 'regeneration -> N)
+    *
+    * >>> val putDoctorResult = Scanamo.putAll(client)("doctors")(
+    * ...   List(Doctor("McCoy", 9), Doctor("Ecclestone", 10), Doctor("Ecclestone", 11)))
+    * >>> Scanamo.getAll[Doctor](client)("doctors")(('actor and 'regeneration) -> List("McCoy" -> 9, "Ecclestone" -> 11))
```

Comment:
I don't really like this syntax, but it was the least bad option I could think of.

I'd love to hear of better alternatives.